vsphere/internal/vmworkflow/virtual_machine_customize_subresource.go

```diff
@@ -79,19 +79,24 @@ func VirtualMachineCustomizeSchema() map[string]*schema.Schema {
 				"domain": {
 					Type:        schema.TypeString,
 					Required:    true,
-					Description: "The FQDN for this virtual machine.",
+					Description: "The domain name for this virtual machine.",
 				},
 				"host_name": {
 					Type:        schema.TypeString,
 					Required:    true,
-					Description: "The host name for this virtual machine.",
+					Description: "The hostname for this virtual machine.",
 				},
 				"hw_clock_utc": {
 					Type:        schema.TypeBool,
 					Optional:    true,
 					Default:     true,
 					Description: "Specifies whether or not the hardware clock should be in UTC or not.",
 				},
+				"script_text": {
+					Type:        schema.TypeString,
+					Optional:    true,
```

Comment:
Similar to `windows_sysprep_text` can we make this field `Sensitive: true`

Reply:
Added!